features/vmware/file.include/etc/cloud/cloud.cfg.d/99-DataSourceVMwareGuestInfo.cfg

```diff
@@ -0,0 +1,13 @@
+# Cloud-Init Datasource for VMware Guestinfo
+#
+# Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+
+datasource_list: [ "VMwareGuestInfo" ]
```

Comment:
This breaks cloud-init for our existing landscape. This does fix it:

```
datasource_list: [ "VMwareGuestInfo", "OVF" ]
```

Can you please add this.

Reply:
After applying the change suggested by @marwinski, another problem occurs:
```
No check method 'dscheck_VMwareGuestInfo' for datasource 'VMwareGuestInfo'
```

The cause is the filename `features/vmware/file.include/usr/bin/dscheck_VMwareGuestInfo.sh`. It must be provided without the suffix `.sh`
see also https://github.com/canonical/cloud-init/blob/fbcb224bc12495ba200ab107246349d802c5d8e6/tools/ds-identify#L1686-L1687

Reply:
With both changes the image works both for our existing landscape and the staging vSphere in the lab.